src/abstract-chart.js

```diff
@@ -99,15 +99,14 @@ class AbstractChart extends Component {
       height,
       paddingTop,
       paddingRight,
-      yLabelsOffset = 12,
       horizontalLabelRotation = 0
     } = config;
     const decimalPlaces =
       this.props.chartConfig.decimalPlaces === undefined
         ? 2
         : this.props.chartConfig.decimalPlaces;
     const yAxisLabel = this.props.yAxisLabel || "";
-
+    const yLabelsOffset = this.props.yLabelsOffset || 12;
```

Comment:
why did you move `yLabelsOffset` from the `chartConfig` into its own prop? that's a potentially breaking api change.

Reply:
AFAIK `yLabelsOffset` is not being used in the code from chart config. `renderHorizontalLabels` function accepts the config prop which has the value 
`{
      width,
      height,
      verticalLabelRotation,
      horizontalLabelRotation
    }` in `line-chart.js line no 277`.

Reply:
it's used on line 123 of abstract chart

Reply:
@Hermanya I have moved it inside `chartConfig`.